### PR TITLE
edit Eptading.md

### PR DESCRIPTION
**Pull request by DerekRoy**
Make sure you have checked _all_ steps below.

### JIRA
- [X] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "[AIRFLOW-XXX] My Airflow PR"
    - https://issues.apache.org/jira/browse/AIRFLOW-XXX

### Description
- [x] Here are some details about my PR, including screenshots of any UI changes:
There were misspelling, word usage errors, and pronoun misuse that have been cleaned up. 

### Tests
- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
There is no need for testing as it is a proof heck and rewrite of the Update text document.

### Commits
- [x ] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

- [ x] Passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

**Comment by jgao54** [comment]
@DerekRoy Thanks for the clean up! Merged #3189 first which resolved in this conflict, my apologies, please resolve so I can merge this.

File: UPDATING.md [hunk 1]
```diff
@@ -250,7 +250,7 @@ dags_are_paused_at_creation = False
 ### Airflow Context variable are passed to Hive config if conf is specified
 
 If you specify a hive conf to the run_cli command of the HiveHook, Airflow add some
-convenience variables to the config. In case your run a sceure Hadoop setup it might be
+convenience variables to the config. In case you run a sceure Hadoop setup it might be
```
Comment (r39132) [inline UPDATING.md:253]: `sceure ` --> `secure `

File: UPDATING.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ to 2.2.0 or greater.
 
 ### Google cloud connection string
 
-With Airflow 1.9 or lower there where two connection strings for the Google Cloud operators, both `google_cloud_storage_default` and `google_cloud_default`. This can be confusing and therefore the `google_cloud_storage_default` connection id has been replaced with `google_cloud_default` to make the connection id consistent across Airflow.
+With Airflow 1.9 or lower there were two connection strings for the Google Cloud operators, both `google_cloud_storage_default` and `google_cloud_default`. This can be confusing and therefore the `google_cloud_storage_default` connection id has been replaced with `google_cloud_default` to make the connection id consistent across Airflow.
```
Comment (r39132) [inline UPDATING.md:63]: `With Airflow 1.9 or lower there were` --> `With Airflow 1.9 or lower, there were`

File: UPDATING.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 
 ## Airflow Master
 
 ### New Webserver UI with Role-Based Access Control
 
-Our current webserver UI uses the Flask-Admin extension. The new webserver UI uses the [Flask-AppBuilder (FAB)](https://github.com/dpgaspar/Flask-AppBuilder) extension. It has built-in authentication support and Role-Based Access Control (RBAC), which provides configurable roles and permissions for individual users.
+The current webserver UI uses the Flask-Admin extension. The new webserver UI uses the [Flask-AppBuilder (FAB)](https://github.com/dpgaspar/Flask-AppBuilder) extension. FAB has built-in authentication support and Role-Based Access Control (RBAC), which provides configurable roles and permissions for individual users.
 
-To turn on this feature, in your airflow.cfg file, under [webserver], set configuration variable `rbac = True`, and then run `airflow` command, which will generate the `webserver_config.py` file in your $AIRFLOW_HOME.
+To turn on this feature, in your airflow.cfg file (under [webserver]) set configuration variable `rbac = True`, and then run `airflow` command, which will generate the `webserver_config.py` file in your $AIRFLOW_HOME.
```
Comment (r39132) [inline UPDATING.md:12]: `To turn on this feature, in your airflow.cfg file (under [webserver]) set configuration variable` --> `To turn on this feature, in your airflow.cfg file (under [webserver]), set configuration variable`

File: UPDATING.md [hunk 4]
```diff
@@ -138,7 +138,7 @@ Type "help", "copyright", "credits" or "license" for more information.
 
 #### Template path of the file_task_handler
 
-The `file_task_handler` logger is more flexible. You can change the default format, `{dag_id}/{task_id}/{execution_date}/{try_number}.log` by supplying Jinja templating in the `FILENAME_TEMPLATE` configuration variable. See the `file_task_handler` for more information.
+The `file_task_handler` logger is more flexible. The default format can be changed, `{dag_id}/{task_id}/{execution_date}/{try_number}.log` by supplying Jinja templating in the `FILENAME_TEMPLATE` configuration variable. See the `file_task_handler` for more information.
```
Comment (r39132) [inline UPDATING.md:141]: `The `file_task_handler` logger is more flexible` --> `The `file_task_handler` logger has been made more flexible`

File: UPDATING.md [hunk 5]
```diff
@@ -225,8 +225,8 @@ indefinitely. This is only available on the command line.
 After how much time should an updated DAG be picked up from the filesystem.
 
 #### dag_dir_list_interval
-How often the scheduler should relist the contents of the DAG directory. If you experience that while developing your
-dags are not being picked up, have a look at this number and decrease it when necessary.
+How often the scheduler should relist the contents of the DAG directory. If while developing your
```
Comment (r39132) [inline UPDATING.md:228]: `How often the scheduler should relist the contents of the DAG directory. If while developing your
+dags they are not being picked up, have a look at this number and decrease it when necessary.` --> `The frequency with which the scheduler should relist the contents of the DAG directory. If while developing
+dags, they are not being picked up, have a look at this number and decrease it when necessary.`

**Review by r39132 — CHANGES_REQUESTED** [review]

**Comment by r39132** [comment]
@DerekRoy Thanks for taking this up. I've requested a few more changes. Please make them and squash your commits into one. Please also create a tracking JIRA for this -- you must follow : https://cwiki.apache.org/confluence/display/AIRFLOW/Contributors%27+Guide

**Comment by r39132** [comment]
Also, please rebase your commit.

**Comment by r39132** [comment]
@DerekRoy ... any updates? this PR will be closed in a week if there is no activity

**Comment by Fokko** [comment]
I would like to get this in, please rebase onto master to resolve the conflicts.

**Comment by r39132** [comment]
@DerekRoy I will be closing this soon if there is no further movement.. and will make the document changes myself. Please respond. Thx.

**Comment by r39132** [comment]
Closing this for inactivity!

**Comment by r39132** [comment]
This is now tracked in https://github.com/apache/incubator-airflow/pull/3248

**Pull request closed without merging**